packages/flutter/lib/src/material/drawer.dart

```diff
@@ -264,10 +281,28 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
   final ColorTween _color = new ColorTween(begin: Colors.transparent, end: Colors.black54);
   final GlobalKey _gestureDetectorKey = new GlobalKey();
 
+  AlignmentDirectional get drawerStartAlignment {
```

Comment:
Should be private (meaning start with an `_`)

Reply:
Fixed